### PR TITLE
feat: Add awesome_bot as a GitHub Action

### PR DESCRIPTION
**Pull request by robsliwi**
Solves https://github.com/reHackable/awesome-reMarkable/issues/190

### CODE REVIEW

**Comment by robsliwi** [comment]
Just for sake of completeness here a CI run: https://github.com/robsliwi/awesome-reMarkable/actions/runs/7389317762/job/20101886841?pr=1

It reveals that https://github.com/reHackable/awesome-reMarkable/blob/master/README.md?plain=1#L132 is reporting a 404 as it's missing a character in the end.

https://github.com/robsliwi/awesome-reMarkable/pull/1/commits/06024d98dd3d8eaa971ea3eee1329c53ab24641d fixes the broken link and introduces the newly green CI :green_heart:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Ruby
```
Comment (Eeems) [inline .github/workflows/ci.yml:1]: This should probably be changed to something more meaningful, like `Verify links`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
```
Comment (Eeems) [inline .github/workflows/ci.yml:7]: We probably don't need to run this on PRs against any branch other than the main one.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ '*' ]
```
Comment (Eeems) [inline .github/workflows/ci.yml:5]: We probably don't need to run this on any branch other than the main one.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
```
Comment (Eeems) [inline .github/workflows/ci.yml:10]: We could probably rename this to `check` or something similar.

**Review by Eeems — CHANGES_REQUESTED** [review]
Just some minor nitpicks. Thank you for tackling this!

**Review by Eeems — APPROVED** [review]

**Pull request merged**